src/status_im/ui/screens/chat/utils.cljs

```diff
@@ -12,22 +12,23 @@
             [status-im.utils.http :as http]))
 
 (defn format-author [from style name]
-  (cond
-    (ens/is-valid-eth-name? name)
-    [react/text {:style {:color colors/blue :font-size 13 :font-weight "500"}}
-     (str "@" (or (stateofus/username name) name))]
-    name
-    [react/text {:style {:color colors/blue :font-size 13 :font-weight "500"}}
-     name]
-    :else
-    [react/text {:style {:color colors/gray :font-size 12 :font-weight "400"}}
-     from]))
+  (let [additional-styles (style false)]
```

Comment:
sorry one last thing are you sure this is actually necessary? what is this argument for? I can't find anywhere where it is true, I'm wondering if we couldn't remove the whole `chosen?` param in these styles fns

Reply:
Yeah it doesn't seem like it's used anywhere but I wasn't completely confident of that (given my inexperience with the codebase) so I made the smallest change I could. It certainly does seem like we could remove the `chosen?` parameter, want me to go ahead and do that?

Reply:
@flexsurfer

Reply:
Il out thid week ill let @flexsurfer decide